ml-agents/mlagents/trainers/ppo/policy.py

```diff
@@ -29,24 +30,42 @@ def __init__(self, seed, brain, trainer_params, is_training, load):
 
         reward_signal_configs = trainer_params["reward_signals"]
 
+        self.devices = get_devices(multi_gpu)
```

Comment:
Rather than passing `multi_gpu` here if you list all the GPUs you can just say `if len(self.gpus) > 1 and multi_gpu: [...]` below.